packages/block-editor/src/components/link-control/style.scss

```diff
@@ -22,6 +22,20 @@ $preview-image-height: 140px;
 		width: 90vw;
 		max-width: $modal-min-width;
 	}
+
+	.show-icon-labels & {
+		.components-button.has-icon {
+			// Hide the button icons when labels are set to display...
+			svg {
+				display: none;
+			}
+			// ... and display labels.
+			// Uses ::before as ::after is already used for active tab styling.
+			&::before {
+				content: attr(aria-label);
+			}
+		}
+	}
 }
```

Comment:
@tellthemachines In a followup it feels like we should make this standardised CSS somehow. 

@ciampo This CSS allows us to use text alternatives for icon buttons. Is it possible to bake this into the component in `@wordpress/components`?

Reply:
Sure! Would you mind opening a separate issue to explain exactly the request? It will make it easier for us (or other folks) to work on it.

Thank you 🙏

Reply:
Done https://github.com/WordPress/gutenberg/issues/48272